services/pingpong/pingpong-status.service.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService, InvalidParameter } = require('..')
+
+const schema = Joi.object({
+  status: Joi.string().required(),
+}).required()
```

Comment:
I know that the we've changed to a different schema (`status` vs `message` + `color`) which is good, though my prior point about utilizing the schema for validation still holds true. I'd like to see the `status` value here in the schema restricted to the known/supported statuses, or if we're going to keep this wide open to accept any string then we need to fail loudly (throw an `InvalidResponse`) in the default switch case in the render function.

Reply:
Sounds good. In order to keep the checks DRY, the render function now throws an error.